test/cmd/basicresources.sh

```diff
@@ -249,9 +249,9 @@ os::cmd::expect_success 'oc create -f test/integration/testdata/test-service.jso
 os::cmd::expect_failure 'oc expose service frontend --create-external-load-balancer'
 os::cmd::expect_failure 'oc expose service frontend --port=40 --type=NodePort'
 os::cmd::expect_success 'oc expose service frontend --path=/test'
-os::cmd::expect_success_and_text "oc get route frontend --output-version=v1 --template='{{.spec.path}}'" "/test"
-os::cmd::expect_success_and_text "oc get route frontend --output-version=v1 --template='{{.spec.to.name}}'" "frontend"           # routes to correct service
-os::cmd::expect_success_and_text "oc get route frontend --output-version=v1 --template='{{.spec.port.targetPort}}'" ""
+os::cmd::expect_success_and_text "oc get route.v1.route.openshift.io frontend --template='{{.spec.path}}'" "/test"
```

Comment:
@deads2k @soltysh this is just to unstuck us for rebase or this is required now?

Reply:
> @deads2k @soltysh this is just to unstuck us for rebase or this is required now?

it's how you specify a specific version (output-version is dead), but something appears to have broken with it...